code/ui/blocks/src/blocks/Controls.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Controls } from './Controls';
+import * as ExampleStories from '../examples/ControlsParameters.stories';
+
+const meta: Meta<typeof Controls> = {
+  component: Controls,
+  parameters: {
+    relativeCsfPaths: ['../examples/ControlsParameters.stories'],
+  },
+};
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {};
+
+export const OfStory: Story = {
+  args: {
+    of: ExampleStories.NoParameters,
+  },
+};
+
+// NOTE: this will throw with no of prop
+export const OfStoryUnattached: Story = {
+  parameters: { attached: false },
+  args: {
+    of: ExampleStories.NoParameters,
+  },
+};
+
+export const IncludeProp: Story = {
```

Comment:
do you think it's overkill to add stories that has both props and parameters to ensure props take priority?

Reply:
I'm not sure. We might have to do a lot of them, theoretically. WDTY?

Reply:
Created a ticket for this.